_sass/_videos.scss

```diff
@@ -1,55 +1,68 @@
 /** 
+ * responsive margins
+ */
+.container {
```

Comment:
I thought that bootstrap tok care of these margins for us already.  I just don't want to repeat work.  I'll take a look later.